.chloggen/alpha_jsonlog.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: new_component
```
Since this isn't being shipped with contrib yet, I assume you're planning on adding it to the contrib distribution? If so, we can say mark it as a new component, unless I'm missing something.

---

I am not sure how much that matters. This is just going to a different section of CHANGELOG.

---

Sure, but that's the point of these fields right? It's not a big deal, I just think this change makes more sense being labelled as a new component vs. enhancement 🤷

---

I'll leave it up to you, I saw a lot of these extensions were moving to `alpha`, so I realize this was part of a larger effort 👍